src/bp-groups/bp-groups-template.php

```diff
@@ -4368,6 +4368,28 @@ function bp_get_group_join_button( $group = false ) {
 		return bp_get_button( apply_filters( 'bp_get_group_join_button', $button, $group ) );
 	}
 
+if ( ! function_exists( 'bb_group_btn_label_text_make_grammatical' ) ) {
+	/**
+	 * Make Correct grammaticle group label.
+	 *
+	 * @param string $full_text  Label of Group User type.
+	 * @param int    $group_id   Group Id
+	 * @param string $label_text Label of Group User type
+	 * @return string
+	 */
```

Comment:
@kishanjasani add since tag in doc block.

Reply:
Add since tag in doc comment.